### PR TITLE
Use install target to put config stuff in correct spots

### PR DESCRIPTION
**Pull request by mulkieran**

### CODE REVIEW

File: stratisd_nonrust.sh [hunk 1]
```diff
@@ -22,6 +22,8 @@ fi
 cd $WORKSPACE
 rustup default 1.49.0
 cargo clean
+cargo install
```
Comment (jbaublitz) [inline stratisd_nonrust.sh:25]: Should this be make install?

Comment (mulkieran) [reply stratisd_nonrust.sh:25]: Whoops!

**Review by jbaublitz — APPROVED** [review]

**Comment by bgurney-rh** [comment]
On a new machine, I'm getting this error in each of the three scripts:
```
install: cannot stat 'docs/stratisd.8': No such file or directory
```

**Comment by jbaublitz** [comment]
@bgurney-rh I just emailed @mulkieran about this. I've resolved it in stratis-storage/stratisd#2488.

**Review by jbaublitz — APPROVED** [review]

**Comment by mulkieran** [comment]
Superceded.

**Pull request closed without merging**